iothub/device/samples/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -101,21 +165,75 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
                 await SendTemperatureAsync(Thermostat1, cancellationToken);
                 await SendTemperatureAsync(Thermostat2, cancellationToken);
                 await SendDeviceMemoryAsync(cancellationToken);
+                await SendDeviceHealthTelemetryAsync(Thermostat1, cancellationToken);
 
                 temperatureReset = _temperature[Thermostat1] == 0 && _temperature[Thermostat2] == 0;
-                await Task.Delay(5 * 1000);
+                await Task.Delay(5 * 1000, CancellationToken.None);
```

Comment:
why specify the ct at all?

Reply:
net5 complains if we don't (either supply the API with a cancellation token or explicitly pass in none).